biogrid/libs/biogrid-simulator/src/lib/biobattery/biobattery-class.ts

```diff
@@ -74,11 +77,11 @@ export class BioBattery implements Battery {
     return validate(batteryValidator);
   }
 
-  getCurrentBatteryEnergy(): Energy {
+  getEnergyInJoules(): Energy {
```

Comment:
why are we renaming this? getters should be named getVariableName

Reply:
This is used in the `biobrain` algorithm in method `determineSupplyingPath` to find the energyProvided.
I changed the getter name so that I don't have to check which gridItem it is since all are expected to have this method getEnergyInJoules

Reply:
If you change the getter name, you should change the class varialbe name as well.